v2/guestbook/main.go

```diff
@@ -190,7 +190,8 @@ func getPrimaryTone(value string, headers http.Header) (tone string) {
 	client := &http.Client{}
 	req, err := http.NewRequest("POST", "http://analyzer:80/tone", b)
 	if err != nil {
-		return "Error talking to tone analyzer service: " + err.Error()
+		return getDummyTone(value)
+		//return "Error talking to tone analyzer service: " + err.Error()
```

Comment:
Let's make sure user sees from guestbook UI the value is returned from dummy tone service as unable to talk to watson.